<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Receiver</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Read status</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="0" column="0" rowspan="6">
           <widget class="QTextEdit" name="edit_multple_comand_rec"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>send comand</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="but_rec_Send_all">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Transmitter</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="ed_tr"/>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QTextEdit" name="edit_multyple_comand_Tr"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="transmitterCmd"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="transmitter_but">
            <property name="text">
             <string>send comand</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="but_readTrStatus">
            <property name="text">
             <string>read tr status</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="but_tr_sendAll">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_Data"/>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButton_Data">
            <property name="text">
             <string>Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
